### ISSUE TITLE
font weights stop working with custom fonts after updating from 0.6.11 to 0.7.0 or later

### ISSUE THREAD

**Issue by DeFUCC** [windicss/vite-plugin-windicss#84]
**Describe the bug**
I use local woff and woff2 fonts via @font-face created with https://google-webfonts-helper.herokuapp.com/fonts/commissioner?subsets=latin  There's two weights defined for one font. I works fine with windi plugin 0.6.11 but after upgrade only the last definition of the font is used and the first is just ignored and falls to the default.

**Minimal Reproductions**
for now can just give the css code that I'm using
```
/* commissioner-regular - latin_cyrillic-ext_cyrillic */
@font-face {
  font-family: 'Commissioner';
  font-style: normal;
  font-weight: 400;
  src: local(''),
    url('./fonts/commissioner-v3-latin_cyrillic-ext_cyrillic-regular.woff2')
      format('woff2'),
    /* Chrome 26+, Opera 23+, Firefox 39+ */
      url('./fonts/commissioner-v3-latin_cyrillic-ext_cyrillic-regular.woff')
      format('woff'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
}

/* commissioner-700 - latin_cyrillic-ext_cyrillic */
@font-face {
  font-family: 'Commissioner';
  font-style: normal;
  font-weight: 700;
  src: local(''),
    url('./fonts/commissioner-v3-latin_cyrillic-ext_cyrillic-700.woff2')
      format('woff2'),
    /* Chrome 26+, Opera 23+, Firefox 39+ */
      url('./fonts/commissioner-v3-latin_cyrillic-ext_cyrillic-700.woff')
      format('woff'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
}
```

**Versions**
- vite-plugin-windicss: works fine on 0.6.11 and breaks from 0.7.0
- vite: 2.0.5
- framework(vue/react/svelte/etc): vue

**Additonal Context**

**Comment by DeFUCC** [comment]
I've noticed the `virtual:` add to css import, but adding it doesn't help with this issue.

**Comment by hannoeru** [comment]
Fixed in v0.8.3
